src/xp.runner/Command.cs

```diff
@@ -80,7 +86,7 @@ public virtual int Execute(CommandLine cmd, ConfigSource configuration)
                 { "date.timezone", new string[] { TimeZoneInfo.Local.Olson() ?? Environment.GetEnvironmentVariable("TZ") ?? "UTC" } },
                 { "extension", configuration.GetExtensions(runtime) }
             };
-            var use = configuration.GetUse() ?? UseComposer();
+            var use = UseFor(cmd) ?? configuration.GetUse() ?? UseComposer();
```

Comment:
Before:

 ```bash
$ xp test.script.php
# Use path: first-to-exist[
#  $USE_XP
#  xp.ini:use_xp
#  ./vendor/xp-framework/core
#  $APPDATA/Composer/vendor/xp-framework/core
#]
```

After:

 ```bash
$ xp test.script.php
# Use path: first-to-exist[
#  $USER/test/vendor/xp-framework/core
#  $USE_XP
#  xp.ini:use_xp
#  ./vendor/xp-framework/core
#  $APPDATA/Composer/vendor/xp-framework/core
#]
```

Reply:
Because typically `$USER/test/vendor/xp-framework/core` will not exist, this change will not affect current scripts. If it *does* exist, then the XP Framework will be loaded from there but the rest won't *until* the user upgrades to XP 9.9